.changelog/DEV-1802.yaml

```diff
@@ -0,0 +1,16 @@
+# Type of change
+change_type: enhancement
+
+# The name of the component
+component: list_versions
+
+# (Optional) One or more tracking issues or pull requests related to the change
+issues: []
+
+# A brief description of the change.
+note: "update Feature's & FeatureList's `list_versions` method"
```

Comment:
Would be good if the note can provide a bit more details on the changes.

Reply:
Will update this part to include info like adding `is_default` column. Any other things that you wish to include on the note?